packages/next/pages/_document.tsx

```diff
@@ -236,6 +239,22 @@ export class Head extends Component<
         ))
   }
 
+  makeStylesheetInert(node: ReactNode): ReactNode {
```

Comment:
readability nit: is `makeStyleSheetInert` actually taking a node as an argument, or an array of children? it's a little confusing, since React.Children.map can work on either

Reply:
ReactNode itself is defined as either. A single element or an array. So this is intentional as the function can work with both.